packages/office-addin-debugging/src/commands.ts

```diff
@@ -7,6 +7,47 @@ import * as devSettings from "office-addin-dev-settings";
 import { AppType, parseAppType, parseDebuggingMethod, startDebugging } from "./start";
 import { stopDebugging } from "./stop";
 
+function getManifestPath(manifestPath: string, appTypeToDebug: string | undefined, prod: boolean): string {
+    let manifestPathToDebug: string = manifestPath;
+    try {
+        if (appTypeToDebug === undefined) {
```

Comment:
It appTypeToDebug : undefined is not supported, why do we have it as an allowed type to the function?